extensions/amp-viewer-integration/0.1/highlight-handler.js

```diff
@@ -157,10 +156,17 @@ export class HighlightHandler {
     /** @private {?Array<!Element>} */
     this.highlightedNodes_ = null;
 
-    if (
-      'fragmentDirective' in document &&
-      isExperimentOn(ampdoc.win, 'use-text-fragments-for-highlights')
-    ) {
+    const platform =
+      /* @type {!./service/platform-impl.Platform} */ Services.platformFor(
+        this.ampdoc_.win
+      );
+
+    // Chrome 81 added support for text fragment proposal. However, it is
+    // not supported across iframes but Chrome 81 thru 92 report
+    // `'fragmentDirective' in document = true` which breaks feature detection.
+    // Chrome 93 supports the proposal that works across iframes, hence this
+    // version check.
+    if (platform.isChrome() && platform.getMajorVersion() >= 93) {
```

Comment:
Can we also include the feature detection? After a bit we can remove the Chromium 93 hard coding and rely solely on feature detection.

Reply:
Done